pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/PythonExtension.groovy

```diff
@@ -129,6 +133,10 @@ class PythonExtension {
         return details
     }
 
+    public void details(Closure cl) {
```

Comment:
This is the correct pattern (ref: https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#dependencies(groovy.lang.Closure) ). You should call it the name you want in the DSL and it accepts a Closure.